.circleci/config.yml

```diff
@@ -113,6 +145,57 @@ jobs:
             source venv/bin/activate
             ./script/typing/check $SCRIPTS
 
+  run-comp-match:
+    <<: *parametrised-webots-executor
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install dependencies
+          command: |
+            # Use deadsnakes to get Python 3.7
+            echo deb http://ppa.launchpad.net/deadsnakes/ppa/ubuntu focal main > /etc/apt/sources.list.d/deadsnakes.list
+            echo deb-src http://ppa.launchpad.net/deadsnakes/ppa/ubuntu focal main >> /etc/apt/sources.list.d/deadsnakes.list
+            apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F23C5A6CF475977595C89F51BA6932366A755776
+
+            export DEBIAN_FRONTEND=noninteractive
+            apt-get update
+            apt-get install --yes zip python3.7{,-dev,-venv}
+
+      - run:
+          name: Prepare team code archive
+          command: |
+            mkdir archives
+            cd archives
+            sed -r 's/# sleep(.*)/print("sleep\1")/' ../controllers/example_controller/example_controller.py > robot.py
```

Comment:
What's this line for?

Reply:
That converts the comments about `sleep for X seconds` into print statements so that there is some output to gauge if the simulator is running or frozen.

Reply:
4ff1d74